workos/events.py

```diff
@@ -57,15 +70,11 @@ def list_events(
             dict: Events response from WorkOS.
         """
 
-        if limit is None:
-            limit = RESPONSE_LIMIT
-            default_limit = True
-
-        params = {
+        params: EventsListFilters = {
             "events": events,
             "limit": limit,
             "after": after,
-            "organization_id": organization_id,
+            "organization_id": organization,
```

Comment:
Right now our other [API methods](https://github.com/workos/workos-python/blob/79a5de376d4645e28a3005356a15fe1a61eb35b1/workos/organizations.py#L116) just use `organization`.

I'm conforming for consistency. Happy to have a cleanup PR sweep through and change all of those params to `organization_id`